Harmony/Tools/CodeMatch.cs

```diff
@@ -125,6 +125,107 @@ internal bool Matches(List<CodeInstruction> codes, CodeInstruction instruction)
 			return true;
 		}
 
+		/// <summary>Creates a code match for local loads</summary>
+		/// <param name="useAddress">Whether to match for address loads</param>
+		/// <param name="name">An optional name</param>
+		/// <returns></returns>
+		public static CodeMatch LocalLoad(bool useAddress = false, string name = null)
+		{
+			CodeMatch match = new CodeMatch(null, null, name);
+
+			if (useAddress)
+			{
+				match.opcodes.AddRange(new[]
+				{
+					OpCodes.Ldloca_S,
+					OpCodes.Ldloca
+				});
+			}
+			else
```

Comment:
In general, I prefer a style where an early return replaces an else unless it really gets weird.